### ISSUE TITLE
Derivative calculator in calculus section 

### ISSUE THREAD

**Issue by SabariGanesh-K** [Tech-N-Science/FunwithScience#408]
**Is your feature request related to a problem? Please describe.**
No. I want to add differentiation calculator in integration section.

**Describe the solution you'd like**
I would like to add derivative calculator in integration section

**Additional context**

**Comment by AnIkeT126** [comment]
Can I contribute on this issue as a part of GSSoC'22

**Comment by rohitkhicher001** [comment]
Can i work on this issue as a gssoc22 contributer.

**Comment by Amit366** [comment]
@rohitkhicher001 yes u can work

**Comment by rohitkhicher001** [comment]
Just wanted to confirm what the issue is. So I have to add differentiation calculator in this https://technscience.com/integration page right?

**Comment by Amit366** [comment]
@rohitkhicher001 yes and also remember that the content should not be plagiarized and also add steps to the calculator

**Comment by Amit366** [comment]
@rohitkhicher001 any update?

**Comment by rohitkhicher001** [comment]
I couldn't do it so please unassign me.

**Comment by RohitSingh107** [comment]
I would like to implement differentiation, integration as well as ordinary differential equations.

Please assign these tasks to me.

**Comment by Amit366** [comment]
@RohitSingh107 r u interested?

**Comment by RohitSingh107** [comment]
Yeah, I can do this .

**Comment by Amit366** [comment]
@RohitSingh107 u can work
